source/_components/light.mqtt.markdown

```diff
@@ -65,6 +65,10 @@ command_topic:
   description: The MQTT topic to publish commands to change the switch state.
   required: true
   type: string
+brightness_command_template:
+  description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to compose message which will be sent to `brightness_command_topic`. Available variables: `brightness`."
```

Comment:
Don't link to /docs/configuration/templating/#processing-incoming-data here, it's confusing, just link to /docs/configuration/templating/